### PR TITLE
Only show future communal repairs

### PR DESCRIPTION
**Pull request by Bhavesh-Patel**

### CODE REVIEW

**Comment by Bhavesh-Patel** [comment]
[🐛 Elapsed and same day communal appointments are displayed](https://trello.com/c/SxliHaNY/326-%F0%9F%90%9B-elapsed-and-same-day-communal-appointments-are-displayed)

File: HousingRepairsOnlineApi.Tests/HelpersTests/RepairQueryHelperTests.cs [hunk 1]
```diff
@@ -1,115 +1 @@
...
-
-            // Assert
-            containerMock.Verify(m => m.Container.GetItemQueryIterator<Repair>(
-                It.Is<QueryDefinition>(u =>
-                    u.GetQueryParameters().Count > 0),
-                It.IsAny<string>(), null));
-        }
-    }
-}
+﻿
```
Comment (markWarr) [inline HousingRepairsOnlineApi.Tests/HelpersTests/RepairQueryHelperTests.cs:1]: This seems to be an empty file

File: HousingRepairsOnlineApi/Helpers/RepairQueryHelper.cs [hunk 2]
```diff
@@ -12,34 +13,36 @@ public class RepairQueryHelper : IRepairQueryHelper
...
-            cosmosContainer.Container.GetItemQueryIterator<Repair>(GetQueryDefinition(repairType, propertyReference));
+        public FeedIterator<Repair> GetRepairSearchIterator(string repairType, string propertyReference)
+        {
+            var query = cosmosContainer.Container.GetItemLinqQueryable<Repair>()
+                .Where(x =>
+                    x.RepairType.ToUpper() == repairType.ToUpper()
+                    && x.Address.LocationId == propertyReference)
+                .Where(isFutureRepair);
+
+            var result = query.ToFeedIterator();
```
Comment (markWarr) [inline HousingRepairsOnlineApi/Helpers/RepairQueryHelper.cs:28]: I'm just wondering if the conversion to using a feedIterator has preserved the date ordering functionality of the original SQL query:

`ORDER BY c.Time.StartDateTime ASC`

Comment (markWarr) [reply HousingRepairsOnlineApi/Helpers/RepairQueryHelper.cs:28]: And if this is needed or was a 'nice to have'.

Comment (Bhavesh-Patel) [reply HousingRepairsOnlineApi/Helpers/RepairQueryHelper.cs:29]: Restored and also added for tenant/leasehold queries

**Review by markWarr — APPROVED** [review]

**Pull request merged**